src/main/java/uk/gov/di/ipv/cri/common/library/service/AuditEventFactory.java

```diff
@@ -57,6 +57,8 @@ private AuditEventUser createAuditEventUser(
             userInfo = new AuditEventUser();
             userInfo.setSessionId(String.valueOf(sessionItem.getSessionId()));
             userInfo.setUserId(sessionItem.getSubject());
+            userInfo.setClientSessionId(sessionItem.getClientSessionId());
```

Comment:
Update AuditEventFactoryTest.java to cover this new code

Reply:
Added to shouldCreateAuditEventWithContextAndExtensions and shouldCreateAuditEventWithContext.